Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -763,31 +756,6 @@ def early_shutdown(self):
             self.session.notify_shutdown_state("Shutting down IPv8...")
             yield self.ipv8.stop(stop_reactor=False)
 
-        if self.channelcast_db is not None:
```

Comment:
Hmm, interesting that we missed this one. Most likely because `LaunchManyCore.py` is just too complex...